osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -213,6 +222,31 @@ void updateScalingModeVisibility()
             }
         }
 
+        private void updateScreenModeWarning()
+        {
+            if (windowModeDropdown.Current.Value != WindowMode.Fullscreen)
+            {
+                windowModeDropdown.WarningText = GraphicsSettingsStrings.NotFullscreenNote;
+                return;
+            }
+
+            switch (fullscreenCapability.Value)
+            {
+                case FullscreenCapability.Unknown:
+                    if (host.Window is WindowsWindow)
+                        windowModeDropdown.WarningText = "Checking for exclusive fullscreen...";
+                    break;
+
+                case FullscreenCapability.Capable:
+                    windowModeDropdown.WarningText = default;
```

Comment:
Given that this text shows a message while "loading", I'd probably expect this to show a positive message afterwards. ie. "osu! is successfully running in exclusive fullscreen, guaranteeing low latency"

Reply:
Yeah if it's involved it can come later, let's see how long it takes to annoy me.